app/views/maintenance_tasks/runs/_info.html.erb

```diff
@@ -13,14 +13,27 @@
   <div class="block">
     <%= link_to("Download CSV", csv_file_download_path(run)) %>
   </div>
+  <hr/>
 <% end %>
 <% if run.arguments.present? %>
-  <div class="block">
-    <h5>Arguments:</h5>
-    <pre>
-      <% run.arguments.each do |key, value| %>
-        <%= "#{key}: #{value}" %>
-      <% end %>
-    </pre>
+<div class="table-container">
+  <h6 class="title is-6">Arguments:</h6>
+    <table class="table">
+      <tbody>
+        <% run.arguments.each do |key, value| %>
+          <tr>
+            <td><var><%= key %></var></td>
+            <td>
+              <% if value.include?("\n") %>
+                <pre><%= value %><pre>
+              <% else %>
+                <code><%= value %><code>
```

Comment:
I didn't like that empty values were just a grey blob, so I took care of that, also handling the nil case which should not happen but we never know 🤷